### DISCUSSION TITLE
Why are release and dist not being included if Debug ID are included. Why one or the other vs both?

### DISCUSSION THREAD

**Discussion by alexkev** [getsentry/sentry-react-native#3474] [General]
@krystofwoldrich Why are release and dist not being included if Debug ID are included. Why one or the other vs both?

_Originally posted by @alexkev in https://github.com/getsentry/sentry-react-native/pull/3164#discussion_r1427038286_

**Comment by krystofwoldrich** [comment]
Hi,
thank you @alexkev for reposting the question,

The reason is that the mapping of the uploaded artifact to release and dist is not needed anymore. The Debug ID will be used to find the correct artifact to symbolicated the incoming stack traces.

This doesn't mean you will lose your Releases in Sentry. Those are still created and events are linked to them.

**Reply by alexkev** [reply]
Thank you for answering my question! I wish this more clear in the source maps documentation. I spent too much time trying get the source maps to upload when I didn't need to.
